packages/babel-parser/src/plugins/flow.js

```diff
@@ -240,12 +240,13 @@ export default (superClass: Class<Parser>): Class<Parser> =>
       } else if (this.match(tt._var)) {
         return this.flowParseDeclareVariable(node);
       } else if (this.isContextual("module")) {
-        if (this.lookahead().type === tt.dot) {
+        this.next();
```

Comment:
Nit: you can use `this.eatContextual("module")` in the if condition

Reply:
done